netci.groovy

```diff
@@ -59,6 +59,47 @@ def addPushJob(String project, String branch, String os, String configuration)
 };
 
 [true, false].each { isPR ->
+  ["RHEL7.2"].each { os->
+    ["Release", "Debug"].each { configuration ->
+
+      def shortJobName = "${os}_Offline_${configuration}";
+      def contextString = "${os} Offline ${configuration}";
+      def triggerPhrase = "(?i).*test\\W+${contextString}.*";
+
+      def newJob = job(Utilities.getFullJobName(project, shortJobName, isPR)){
+        steps{
+            shell("cd ./source-build;git submodule update --init --recursive");
+            shell("cd ./source-build;./build.sh /p:ArchiveDownloadedPackages=true /p:Configuration=${configuration} ${loggingOptions}");
```

Comment:
Don't we need to pass OfflineBuild=true here?

Reply:
No.  Not on this initial build.  OfflineBuild=true is set in the next statement: ./tarball-output/build.sh